### PR TITLE
feat(Flex.Stack): change default vertical spacing (gap) from small to medium (effects Card with stack as well)

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

**Comment by joakbjerk** [comment]
The distance between the `Toggle.Group` label and `Toggle.Buttons` seems a little large.

Screenshot 2024-04-23 at 08 09 35

**Review by joakbjerk — APPROVED** [review]
Looks much better! 🔥

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx [hunk 1]
```diff
@@ -306,4 +305,4 @@ For changing sizes of e.g. input fields, you may have a look at [Form.Appearance
 
 Eufemia Forms consists of helper components and tools so you can declaratively create interactive form components that flawlessly integrates between existing data and your custom form components. This ensures a common look and feel, even when ready-made components are combined with your local custom components.
 
-Read more about on how to [create your own component](/uilib/extensions/forms/create-component).
+Read more about on how to [create your own component](/uilib/extensions/forms/create-component).import properties from '@dnb/eufemia/src/style/themes/theme-ui/properties'
```
[the remark this answers is not in the document]
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx:308]: The import should not be there 😇

**Pull request merged**